src/react-app/index.css

```
@@ -1,29 +1,41 @@
+  font-display: swap;
+}
+
 :root {
   --black: #0a0f0a;
   --green: #4caf50;
   --red: #b04a4a;
   --light-grey: #7fa87f;
-  --faded-green: #2d4a2d;
+  --faded-green: #346335;
   --accent: #4a9999;
+  --font-terminal: "Glass TTY VT220", "Courier New", Courier, monospace;
```

Stylelint (17.14.0)

[error] Expected "Courier" to be "courier" (value-keyword-case)

(value-keyword-case)